plugins/modules/sonic_static_routes.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2022 Dell Inc. or its subsidiaries. All Rights Reserved
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#############################################
+
+"""
+The module file for sonic_static_routes
+"""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+module: sonic_static_routes
+version_added: 2.0.0
+short_description: Manage static routes configuration on SONiC
+description:
+  - This module provides configuration management of static routes for devices running SONiC
+author: "Shade Talabi (@stalabi1)"
+options:
+  config:
+    type: list
+    elements: dict
+    description:
+      - Manages 'static_routes' configurations
+    suboptions:
+      vrf_name:
+        required: True
+        type: str
+        description:
+          - Name of the configured VRF on the device.
+      static_list:
+        type: list
+        elements: dict
+        description:
+          - A list of 'static_routes' configurations.
+        suboptions:
+          prefix:
+            required: True
+            type: str
+            description:
+              - List of locally configured static routes.
+          next_hops:
+            type: list
+            elements: dict
+            description:
+              - A list of next-hops to be utilised for the static route being specified.
+            suboptions:
+              index:
+                required: True
+                type: dict
+                description:
+                  - An identifier utilised to uniquely reference the next-hop.
+                suboptions:
+                  blackhole:
+                    type: bool
+                    default: False
+                    description:
+                      - Indicates if route is blackholed.
```

Comment:
This description doesn't add much information.

It would be more useful to define the blackhole buzzword.

For example:

```
Indicates that packets matching this route should be discarded.